model/src/form/form-metadata/index.ts

```diff
@@ -30,10 +31,10 @@ export const teamEmailSchema = Joi.string()
   .required()
 
 /**
- * Joi schema for `FormMetadata` interface
- * @see {@link FormMetadata}
+ * Joi schema for `FormMetadataInput` interface
+ * @see {@link FormMetadataInput}
  */
-export const formMetadataCreateSchema = Joi.object<FormMetadata>()
+export const formMetadataInputSchema = Joi.object<FormMetadata>()
```

Comment:
Sorry, we need to align the type too

```suggestion
export const formMetadataInputSchema = Joi.object<FormMetadataInput>()
```

Reply:
Fixed in 97b16a03e5760c938d8c46dfea0320a5d8d44820